test/codeu/chat/server/JSONTest.java

```diff
@@ -1,11 +1,8 @@
 package codeu.chat.server;
 
 import static org.junit.Assert.*;
-import codeu.chat.common.ConversationHeader;
-import codeu.chat.common.ConversationPayload;
-import codeu.chat.common.Message;
-import codeu.chat.common.ServerInfo;
-import codeu.chat.common.User;
+
+import codeu.chat.common.*;
```

Comment:
I generally try to avoid wildcard imports.

https://google.github.io/styleguide/javaguide.html#s3.3-import-statements